tuf/api/metadata.py

```diff
@@ -994,6 +996,13 @@ def __init__(
                 "Only one of the attributes 'paths' and"
                 "'path_hash_prefixes' can be set!"
             )
+
+        if paths is None and path_hash_prefixes is None:
+            raise ValueError(
+                "At least one of the attributes 'paths' and"
+                "'path_hash_prefixes' must be set!"
+            )
```

Comment:
I'd really appreciate if we could squeeze the exceptions and comments to the minimum number of lines that are required to convey the message... in the long term this has a major effect on readability. The content in this one seems a bit misleading as well so maybe: 

```suggestion
            raise ValueError("One of paths or path_hash_prefixes must be set")
```

Reply:
I tried to shorten the exception messages and also make them look consistently. It may have slightly changed the meaning though: 34f59c9